code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -21,6 +21,8 @@
 	var/lip_color = "white"
 
 	var/age = 30		//Player's age (pure fluff)
+	var/height = "average height"
+	var/physique = "average build"
```

Comment:
"average build" looks a little weird when it gets formatted: "with an average build physique!"

I'd suggest changing this option (as well as the choices in the selector) to just say "average", reworking the verbiage across choices in general, and/or changing how build gets phrased in the examine message.